src/net_processing.cpp

```diff
@@ -2594,6 +2594,35 @@ bool static ProcessMessage(CNode* pfrom, const std::string& strCommand, CDataStr
     return true;
 }
 
+static bool SendRejectsAndCheckBan(CNode* pnode, CConnman& connman)
+{
+    AssertLockHeld(cs_main);
+    CNodeState &state = *State(pnode->GetId());
+
+    BOOST_FOREACH(const CBlockReject& reject, state.rejects)
```

Comment:
Nit: avoid `BOOST_FOREACH` if possible (though this was moved code, so perhaps not applicable, but..).
```C++
for (const CBlockReject& reject : state.rejects) [...]
```